### PR TITLE
Four Column Grid are created

### PR DESCRIPTION
**Pull request by 24sint**
#### Background Information
- Four column grid is created, and it's one part of the project.

#### Impact
- The graduates (students) are one of the targeted people in the project.

#### Issue
Solves: https://github.com/team-refcode/refcode.org/issues/148

Screen Shot 2019-09-13 at 7 13 53 PM

### CODE REVIEW

File: components/Molecule/fourColumnGrid.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      margin: 0 auto;
+      grid-template-columns: 1fr 1fr 1fr 1fr;
+    }
+
+    .fourColumnGrid img {
+      width: 100%;
+      margin-bottom: 1em;
+    }
+
+    .fourColumnGrid figcaption {
```
Comment (brentonstrine) [inline components/Molecule/fourColumnGrid.html]: I don't think this needs to be part of the `fourColumnGrid` component.

File: components/Molecule/fourColumnGrid.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      text-align: center;
+      display: grid;
+    }
+
+    .fourColumnGrid figcaption a {
```
Comment (brentonstrine) [inline components/Molecule/fourColumnGrid.html]: I don't think this needs to be part of the `fourColumnGrid` component.

File: components/Molecule/fourColumnGrid.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    <div class="fourColumnGrid">
+      <figure class="col-1">
+        <img src="https://via.placeholder.com/150">
+        <figcaption>
+          <span>Name</span>
+          <a href="#">Website: example.com</a>
+          <a href="#">GitHub Profile</a>
+          <a href="#">LinkedIn Profile</a>
+        </figcaption>
+      </figure>
```
Comment (brentonstrine) [inline components/Molecule/fourColumnGrid.html]: Using `figure` and `figcaption` here is not generic enough. It is specific to the Graduates page. It's best to build the component with something more generic, like this:

```
<div>
    <img src="https://via.placeholder.com/150">
</div>
```

Comment (24sint) [reply components/Molecule/fourColumnGrid.html]: I tried to fix all the feedback if I did in the right way.

**Comment by 24sint** [comment]
@brentonstrine Thanks, Brenton. I came to my home when the time pass at 12:15. I fixed the feedback and applied the column-gap property in the four-column grid. But doesn't work when the column down to two-column for medium screen and one for the small screen in the media query. However, when I used the margin for .col in the media query that become perfect.

**Comment by 24sint** [comment]
Ohh, correction, I used **grid-column-gap** instead of **column-gap** to apply for all level, mean the four-column grid and the media query. No need to use margin.

**Pull request merged**